packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -35,7 +35,7 @@ const SearchPage = props => {
     }
 
     const { products } = data;
-    const { filters, total_count, items } = products;
+    const { filters, items } = products;
```

Comment:
It turns out that `total_count` was actually accurate here, we're just not passing `pageSize` to `productSearch.graphql`, so its falling back on the default value of 20. Given that pagination is missing from this component, easy to see why this wasn't making sense.

It seems preferable to fix the count in the suggestion component, and add pagination to `searchPage`. Given that's a much larger undertaking, I would be fine with fixing the count on suggestion, and creating a Story to add pagination to search.

Reply:
Ah, gotcha - I was going to dig into that as a follow up, thanks.

Reply:
I didn't go with `total_count` because we're not showing that number of items. To a user, there'd still be a discrepancy until that pagination to search ticket is completed.

Reply:
Sure. I'll settle for a comment explaining we're only displaying the first twenty results, and the counts will be updated to be accurate once pagination is added.

Reply:
I may or may not have just gone ahead and implemented Pagination.... https://github.com/magento/pwa-studio/pull/2037/commits/95ab17bdf0473277b1f2c70d39081d0e23720614

😄